src/routes/Routes.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import { Route, useHistory } from 'react-router-dom'
+import Question from '../views/Question'
+import Home from '../views/Home'
+import Login from '../views/Login'
+import Signup from '../views/Signup'
+import AskQuestion from '../views/AskQuestion'
+import About from '../views/About'
+import NotFound from '../views/404'
+import Switch from 'react-bootstrap/esm/Switch'
+import ReactGA from 'react-ga'
+
+export default () => {
+  const history = useHistory()
+  ReactGA.initialize('UA-204918741-1')
+  useEffect(() => {
+    const host = window.location.hostname
+    // To not track activities in GA during development testing
+    if (host !== 'localhost') {
+      trackPageView() // To track the first pageview upon load
+      history.listen(trackPageView) // To track the subsequent pageviews
+    }
+  }, [history])
+
+  function trackPageView () {
+    ReactGA.pageview(window.location.pathname + window.location.search)
+  }
+  return (
+    <Switch>
+      <Route
+        exact
+        key='question'
+        path='/question/:id'
+        component={Question}
+      />,
```

Comment:
You left a `,` after each route here. This used to return an array where it needed the `,` but now you're returning a component.

Reply:
Those `,` are getting output in plain text before the `404` content.

Reply:
ohh good catch! I didn't realize it

Reply:
Good catch ! I didn't realise it